packages/ui5-application-writer/test/data.test.ts

```diff
@@ -113,6 +113,42 @@ describe('Setting defaults', () => {
                 ui5Theme: 'sap_fiori_3',
                 ui5Libs: defaultUI5Libs
             }
+        },
+        {
+            input: {
+                version: 'snapshot-1.80',
+                frameworkUrl: 'https://ui5.unreleased.sap.com'
+            },
+            expected: {
+                framework: 'SAPUI5',
+                frameworkUrl: 'https://ui5.unreleased.sap.com',
+                version: 'snapshot-1.80',
+                localVersion: 'snapshot-1.80',
+                minUI5Version: '1.60.0',
```

Comment:
minUI5Version should be ‘snapshot-1.80’
@zdravko-georgiev correct me if i am wrong.

Reply:
@devinea This is how it was implemented...if `minUI5Version` is not provided it defaults to `1.60` https://github.com/SAP/open-ux-tools/pull/319/files#diff-0f207248944a1732a5b7343b9d1bb29e22bbae5fa2acce7a573a344c04c856d2L72

But I think you are correct, I will update to use the current version provided if not present.

Reply:
@devinea @IainSAP We need to be careful here, If I am not mistaken snapshot versions are not valid for the `manifest.json`.